### ISSUE TITLE
Images fail to embed if using a space in the image file name

### ISSUE THREAD

**Issue by LiquidZulu** [LiquidZulu/liquidzulu.github.io#138]
If an image has a url like `/path/to/some image.png "alt text")` then it fails to be turned into an actual `<img>` tag. I have written a remark plugin that properly URI encodes these paths with [this commit](https://github.com/LiquidZulu/liquidzulu.github.io/compare/source...fix-remark-image-embeds), but it is not being applied properly. I can see that it is indeed correctly mapping `![img](/path/to/some image.png "alt text)` -> `![img](/path/to/some image.png "alt text")` but it is still showing raw markdown, as against embedding that valid image.

I do not know why it is that the aforementioned plugin does not work, but I suspect that it is being applied after the images have already been embedded, but I do not know. Maybe I need to change the type from text to image 🤔.

**Comment by LiquidZulu** [comment]
> I can see that it is indeed correctly mapping ![img](/path/to/some image.png "alt text) -> ![img](/path/to/some image.png "alt text")

What is bro yapping about?
